test/e2e/logs.test.js

```diff
@@ -145,17 +145,17 @@ module.exports = () => {
 
                     expect(res.body.data).to.be.an('array');
                     expect(res.body.data.length).to.be.greaterThan(1);
-                    expect(res.body.data[0].createdAt).to.be.gte(timeFrom);
+                    expect(res.body.data[0].createdAt).to.be.gte(fromTime);
 
                     done();
                 });
         });
 
         it('should support filtering by creation time', (done) => {
-            const timeFrom = 946684800000;
+            const fromTime = 946684800000;
             const timeTo = 1577833200000;
```

Comment:
Change `timeTo` to `toTime` to stay consistent with variables

Reply:
Done